coffeescript-loader/loader.js

```diff
@@ -5,7 +5,10 @@ import { dirname, extname, resolve as resolvePath } from 'path';
 import { fileURLToPath, pathToFileURL } from 'url';
 
 import CoffeeScript from 'coffeescript';
-
+const compile = (source, filename) => CoffeeScript.compile(source, {
```

Comment:
Can we call this `coffeeCompile` or something to distinguish it from `module._compile`?